quotes_recommender/quote_scraper/pipelines.py

```diff
@@ -12,6 +14,9 @@
 class QuotesToQdrantPipeline:
     """Scrapy Quotes Pipeline"""
 
+    with open(TAG_MAPPING_PATH, 'r', encoding='utf-8') as file:
+        tag_mappings = json.load(file)
+
```

Comment:
verwende hier bitte `TXT_ENCODING`. Hat @SvenLigensa die JSON file genauso beim Export encoded? Sonst klappt das nicht...

Reply:
move das mal bitte in `open_spider`